internal/ingress/controller/store/ingress_class.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
why ingress_class on this file and not ingressclass like in the controller?

Reply:
hum, maybe I've changed this before calling it ingressclass.

Can rename :)